.github/workflows/cd.yml

```diff
@@ -112,7 +112,90 @@ jobs:
       pull-requests: write
     steps:
       - uses: release-drafter/release-drafter@v5
+        id: update-release-draft
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           version: ${{ env.VERSION }}
+
+  push-release-binaries:
+    needs: update-release-draft
+    runs-on: ubuntu-22.04
+    defaults:
+      run:
+        working-directory: ./rust
+    strategy:
+      matrix:
+        include:
+          - package: relay
+          - package: gateway
+    env:
+      BINARY_DEST_PATH: ${{ matrix.package }}-${ env.VERSION }}-linux-x86_64
+    steps:
+      - uses: actions/checkout@v4
+      - name: Update toolchain
+        run: rustup show
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: ubuntu-22.04-${{ runner.arch }}
+          workspaces: ./rust
+          save-if: false
+      - name: Build release binaries
+        env:
+          CARGO_OUTPUT_FILE: target/release/${{ matrix.package }}
+        run: |
+          cargo build --release -p ${{ matrix.package }}
```

Comment:
I can compile both packages to statically linked binaries without issues. Should we do that?

Reply:
Yeah I think so. I've updated this to build with `cross-rs` for `aarch64` and `x86_64`. We may want to add `mips-unknown-linux-musl` as well to support embedded devices like home office routers/switches.

[We used `cross-rs` for Wireguardex](https://github.com/firezone/wireguardex/blob/main/.github/workflows/ci.yml) with good results. Open to another suggestion if there's a better way to cross-compile now.